pkgs/applications/window-managers/gamescope/default.nix

```diff
@@ -0,0 +1,91 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, pkg-config
+, ninja
+, xorg
+, libdrm
+, vulkan-loader
+, wayland
+, wayland-protocols
+, libxkbcommon
+, libcap
+, SDL2
+, pipewire
+, udev
+, pixman
+, libinput
+, libseat
```

Comment:
```
error: Function called without required argument "libseat" at /home/jon/.cache/nixpkgs-review/pr-186347-1/nixpkgs/pkgs/applications/window-managers/gamescope/default.nix:18, did you mean "libXext", "libjcat" or "libnet"?
(use '--show-trace' to show detailed location information)
nix --experimental-features nix-command --system x86_64-linux eval --json --impure --expr (import /nix/store/ydf4ra1q1bi9v2xkwdslxj44m9b77km3-nixpkgs-review-2.6.4/lib/python3.9/site-packages/nixpkgs_review/nix/evalAttrs.nix { allowAliases = false; attr-json = /run/user/1000/tmps5f6z53z; }) failed to run, /run/user/1000/tmps5f6z53z was stored inspection
https://github.com/NixOS/nixpkgs/pull/186347 failed to build
```
We shouldn't be using aliased package names

Reply:
Oh I see, hm, so another commit should be done addressing this right? I'm not so sure where to look for the original names the alias refer to. However, whenever that happens the backport will have to be done manually right?

Reply:
This was fixed in another PR witch we could also just backport